docs/help/index.md

```diff
@@ -1,43 +1,31 @@
 ---
 layout: default
-title: Help Intro
+title: Getting Started
```

Comment:
Did you notice that this title appears in the tab of the (Firefox) browser. As a general remark: therefore we might need a shorter one here than below.